scripts/src/utils.ts

```diff
@@ -2,6 +2,12 @@ import * as _path from "path";
 import { join } from "path";
 import * as fs from "fs";
 
+import { Express } from "express";
+import { Context } from "./public/routes";
+export interface RequestWithContext extends Express.Request {
+	context?: Context;
```

Comment:
where are you setting this?

Reply:
This code is already exist, I am just using it here as well.

Reply:
it's probably not the right place to put it then